xsd/netex_part_5/part5_nd/netex_nm_mobilityServiceConstraintZone_support.xsd

```diff
@@ -10,7 +10,10 @@
 				<Contributor>Nicholas Knowles</Contributor>
 				<Contributor>Christophe Duquesne</Contributor>
 				<Coverage>Europe</Coverage>
-				<Creator>First drafted for NeTEx version 1.0 CEN TC278 WG3 SG9 Editor Nicholas Knowles.  mailto:schemer@netex.org.uk</Creator>
+				<Creator>First drafted for NeTEx version 1.0 CEN TC278 WG3 SG9 Editor Nicholas Knowles.  mailto:schemer@netex.org.uk</Creator>allUsesAllowed
```

Comment:
I don't think this suffix is supposed to be there.

Reply:
I edited the files for the 2 typos ... Stefan, I think we can direct do such type fix when we spot them

Reply:
Only if the pull request is editable. And to be frank, you approved merging, while you are now editing the pull request.

Reply:
Yes sure ...making them editable is a good point. This one seam to be editable, that's how I did the correct the 2 issues you spotted and that I missed... thanks for spotting them